packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -5446,16 +5446,27 @@ class SemanticsConfiguration {
   }
 
   /// Whether the owning [RenderObject] can hold the input focus.
+  @Deprecated(
+    'Check if isFocused is null instead. '
+    'This feature was deprecated after v3.34.0-0.0.pre.',
+  )
   bool get isFocusable => _flags.isFocusable;
+
+  @Deprecated(
+    'Setting isFocused automatically set this to true. '
+    'This feature was deprecated after v3.34.0-0.0.pre.',
+  )
   set isFocusable(bool value) {
     _flags = _flags.copyWith(isFocusable: value);
     _hasBeenAnnotated = true;
   }
 
   /// Whether the owning [RenderObject] currently holds the input focus.
-  bool get isFocused => _flags.isFocused;
-  set isFocused(bool value) {
-    _flags = _flags.copyWith(isFocused: value);
+  bool? get isFocused => _flags.isFocusable ? _flags.isFocused : null;
+  set isFocused(bool? value) {
+    if (value != null) {
```

Comment:
if this is null, should this reset isFocusable? I looked at other property looks like they just pass the null to the flags, not sure if those make sense neither